test/e2e/framework/suites.go

```diff
@@ -180,10 +180,10 @@ func gatherTestSuiteMetrics() error {
 		return fmt.Errorf("failed to grab metrics: %v", err)
 	}
 
-	metricsForE2E := (*e2emetrics.MetricsForE2E)(&received)
+	metricsForE2E := (*e2emetrics.ComponentCollection)(&received)
 	metricsJSON := metricsForE2E.PrintJSON()
 	if TestContext.ReportDir != "" {
-		filePath := path.Join(TestContext.ReportDir, "MetricsForE2ESuite_"+time.Now().Format(time.RFC3339)+".json")
```

Comment:
This is changing a file path, will this break anything?

Reply:
No, this change is not mandatory at all.
I just changed here for the consistency because this path seems to come from the name.
It is open to revert here if we want.